docs/use/markdown.md

```diff
@@ -1,27 +1,109 @@
-# Notebooks in markdown
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: '0.8'
+    jupytext_version: '1.4.1'
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+# Notebooks as Markdown
 
-MyST-NB also provides functionality for notebooks within markdown. This lets you
-include and control notebook-like behavior from with your markdown content.
+MyST-NB also provides functionality for writing notebooks in a text-based format,
+utilising the [MyST Markdown format](https://jupytext.readthedocs.io/en/latest/formats.html#myst-markdown) outlined in [jupytext](https://jupytext.readthedocs.io).[^download]
+These files have a 1-to-1 mapping with the notebook, so can be opened as notebooks
+in Jupyter Notebook and Jupyter Lab (with jupytext installed), and are also integrated
+directly into the {ref}`Execution and Caching <execute/cache>` machinery!
 
-The primary way to accomplish this is with the `{jupyter-execute}` directive. The content
-of this directive should be runnable code in Jupyter. For example, the following
-code:
+[^download]: This notebook can be downloaded as
+            **{jupyter-download:notebook}`markdown`** and {download}`markdown.md`
 
-````
-```{jupyter-execute}
+They are distinguished from standard Markdown files by adding this top matter to the first line of you file (or the relevant `kernelspec` for your code):
+
+```md
+---
+jupytext:
+  text_representation:
+    format_name: myst
+kernelspec:
+  display_name: Python 3
+  name: python3
+---
+```
+
+```{tip}
+You can also create the file from an existing notebook: `jupytext notebook.ipynb --to myst`
```

Comment:
I think this needs to be `md:myst`

Reply:
nope, I added in some sneaky logic into jupytext to intercept `myst` and read it as `md:myst`, so both work.

Reply:
Hmm - is this in master and not released yet? I just tried it locally on the latest pypi version and it didn't work

Reply:
Not sure, I using the dev version, so it will definitely work at some point. This conversion won't actually work at the moment with latest myst versions anyway, until mwouts/jupytext#473 is merged and released

Reply:
Note I don't use jupytext as a dependency at present, as some of the code is a replica of the code in mwouts/jupytext#473, which could be swapped in at some point. To keep it simple, we only allow the myst format for now, not any of the other jupytext formats